src/anemoi/training/utils/checkpoint.py

```diff
@@ -7,16 +7,30 @@
 # granted to it by virtue of its status as an intergovernmental organisation
 # nor does it submit to any jurisdiction.
 
-
 from __future__ import annotations
 
+import logging
 from pathlib import Path
 
 import torch
+import torch.nn as nn
 from anemoi.utils.checkpoints import save_metadata
 
 from anemoi.training.train.forecaster import GraphForecaster
 
+LOGGER = logging.getLogger(__name__)
+
+
+def create_new_zip_path(zip_path: Path | str, patch_str: str = "patched") -> Path:
```

Comment:
Sorry, I don't understand the role of this function. Can't a user simply provide the checkpoint name in the config file?

Reply:
Maybe I am missing something, sorry.

Reply:
Hey Matt, talked to @gabrieloks and decided to ignore #57 on this and let the code crush for that issue.
This has been removed!